doc/btp_cap.txt

```diff
@@ -0,0 +1,269 @@
+CAP Service (ID 24)
+===================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<non-controller>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+	Opcode 0x02 - Discover and Subscribe
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	<none>
+
+		This command is used to discover all remote PACS, ASCS and BASS
+		characteristics. In case of an error, the error status response
+		will be returned. In case of a success, the IUT continues processing
+		the command asynchronously. During discovery, the IUT may send events:
+					Discover and Subscribe Completed event
+					Codec Capabilities Found event
+					ASE Found event
+
+	Opcode 0x03 - Ordered Access
+		Controller Index:	<controller id>
+		Command parameters:	Address_Type (1 octet)
+					Address (6 octets)
+		Response parameters:	<none>
+
+		This command is used to execute the CSIP Ordered Access.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing
+		the command asynchronously.
```

Comment:
Hmm, I have mixed feelings about that one, for 2 reasons:
1. It's CSIP procedure, so it should be placed in CSIP
2. The actual procedure is missing. The ordered access performs some procedure on set. It's unclear to me what this BTP command does

Reply:
I created the CSIP BTP and renamed the command to Start Ordered Access.